theme/theme.go

```diff
@@ -11,28 +11,105 @@ import (
 
 const (
 	// ColorRed is the red primary color name
+	//
+	// Since: 1.4.0
 	ColorRed = "red"
 	// ColorOrange is the orange primary color name
+	//
+	// Since: 1.4.0
 	ColorOrange = "orange"
 	// ColorYellow is the yellow primary color name
+	//
+	// Since: 1.4.0
 	ColorYellow = "yellow"
 	// ColorGreen is the green primary color name
+	//
+	// Since: 1.4.0
 	ColorGreen = "green"
 	// ColorBlue is the blue primary color name
+	//
+	// Since: 1.4.0
 	ColorBlue = "blue"
 	// ColorPurple is the purple primary color name
+	//
+	// Since: 1.4.0
 	ColorPurple = "purple"
 	// ColorBrown is the brown primary color name
+	//
+	// Since: 1.4.0
 	ColorBrown = "brown"
 	// ColorGray is the gray primary color name
+	//
+	// Since: 1.4.0
 	ColorGray = "gray"
 )
 
+var (
+	// Colors specifies each of the known colour names that a theme can contain.
+	//
+	// Since 2.0.0
+	Colors = struct {
+		Background     fyne.ThemeColorName
+		Foreground     fyne.ThemeColorName
+		Button         fyne.ThemeColorName
+		DisabledButton fyne.ThemeColorName
+		Disabled       fyne.ThemeColorName
+		Focus          fyne.ThemeColorName
+		Hover          fyne.ThemeColorName
+		PlaceHolder    fyne.ThemeColorName
+		Primary        fyne.ThemeColorName
+		ScrollBar      fyne.ThemeColorName
+		Shadow         fyne.ThemeColorName
+	}{
+		"background",
+		"foreground",
+		"button",
+		"disabledButton",
+		"disabled",
+		"focus",
+		"hover",
+		"placeholder",
+		"primary",
+		"scrollBar",
+		"shadow",
+	}
+
+	// Sizes specifies each of the known size names that a theme can contain.
+	//
+	// Since 2.0.0
+	Sizes = struct {
+		Padding        fyne.ThemeSizeName
+		ScrollBar      fyne.ThemeSizeName
+		ScrollBarSmall fyne.ThemeSizeName
+		Text           fyne.ThemeSizeName
+		InlineIcon     fyne.ThemeSizeName
```

Comment:
nit: ditto - it isn't a public API, but the alphabetical ordering would be nice :)